submissions/misshatsker/document-object-model/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="index.css" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Aboreto&display=swap" rel="stylesheet">
+    <link rel="icon" type="image/x-icon" href="./icons/heart.png">
+    <title>Document-Object-Model-misshatsker</title>
+</head>
+<body>
+    <header class="header">
+        <span class="logo">Facts About Pets</span>
+        <div class="menu-icon"><span class="navicon"></span></div>
+        <ul class="menu"></ul>
+    </header>
+    <main>
+        <div class="full-main-page">
+            <img class="media-animal" alt="">
```

Comment:
Try to fill an alt attribute with an image description. That increases your code accessibility.